src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/steps/data/fetch-form-data.ts

```diff
@@ -0,0 +1,148 @@
+import AppValues from "src/app/common/app.values";
+import { JsonFormData } from "../../add-polling-source-form.types";
+
+export const FETCH_FORM_DATA: JsonFormData = {
+    url: {
+        controls: [
+            {
+                name: "url",
+                label: "Url",
+                value: "",
+                type: "text",
```

Comment:
Everywhere in these data files - why `type` field is not referencing an enum you've defined?

```
export enum ControlType {
    TEXT = "text",
    ARRAY_KEY_VALUE = "array-key-value",
    CHECKBOX = "checkbox",
    ARRAY_KEY = "array-key",
    SCHEMA = "schema",
    EVENT_TIME = "event-time",
    CACHE = "cache",
    ORDER = "order",
    TYPEAHEAD = "typeahead",
}
```

Reply:
Fixed